### PR TITLE
chore(deps): update all dependencies and use webpack 5

### PR DESCRIPTION
**Pull request by ylemkimon**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?
N/A

### Motivation / Use-Case
* Updated all dependencies
  * `prettier`: [improve formatting for assignments of classes with long superclass names](https://prettier.io/blog/2020/11/20/2.2.0.html#improve-formatting-for-assignments-of-classes-with-long-superclass-names-9341httpsgithubcomprettierprettierpull9341-by-sosukesuzukihttpsgithubcomsosukesuzuki)
  * `webpack-merge`: [merge has been moved as a regular import](https://github.com/survivejs/webpack-merge/blob/develop/CHANGELOG.md#503--2020-07-06)
  * `schema-utils`: [the packages exports was changed](https://github.com/webpack/schema-utils/blob/v3.0.0/CHANGELOG.md#300-2020-10-05)
  * `puppeteer`: `waitFor` is deprecated (https://github.com/puppeteer/puppeteer/issues/6214)
* Use webpack 5
  * Set `target: ['web', 'es5']` for client sources

### Breaking Changes
N/A

### CODE REVIEW

File: client-src/clients/SockJSClient.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ const SockJS = require('sockjs-client/dist/sockjs');
 const { log } = require('../default/utils/log');
 const BaseClient = require('./BaseClient');
 
-module.exports = class SockJSClient extends BaseClient {
+module.exports = class SockJSClient extends (
```
Comment (shaodahong) [inline client-src/clients/SockJSClient.js:7]: Cause by editor format?

Comment (ylemkimon) [reply client-src/clients/SockJSClient.js:7]: `prettier` was updated [to improve formatting for assignments of classes with long superclass names](https://prettier.io/blog/2020/11/20/2.2.0.html#improve-formatting-for-assignments-of-classes-with-long-superclass-names-9341httpsgithubcomprettierprettierpull9341-by-sosukesuzukihttpsgithubcomsosukesuzuki).

Comment (shaodahong) [reply client-src/clients/SockJSClient.js:7]: Oh, thanks, weird for this to extra brackets, maybe fix by https://github.com/prettier/prettier/issues/9744

issue link https://github.com/prettier/prettier/pull/9341#issuecomment-732446575

Comment (shaodahong) [reply client-src/clients/SockJSClient.js:7]: Is not important for this pr

**Review by hiroppy — APPROVED** [review]

**Review by evilebottnawi — APPROVED** [review]

**Comment by evilebottnawi** [comment]
Big thanks

**Pull request merged**